Makefile

```diff
@@ -1,23 +1,25 @@
 .DEFAULT: all
 .PHONY: all clean image minikube-publish manifest test kured-all
 
-TEMPDIR=./.tmp
```

Comment:
`.tmp` seems to be still used in other places.

Reply:
I didn't find it in another places. The grep didn't return any result. What else did I miss?